persistence/src/main/java/cz/muni/fi/pa165/dao/BottleDAO.java

```diff
@@ -25,6 +27,14 @@
 
     List<Bottle> findByToxic(boolean toxic);
 
+    @Query("SELECT b FROM Bottle b " +
+            "INNER JOIN b.bottleType bt " +
+            "INNER JOIN bt.manufacturedBy m " +
+            "WHERE bt.id=b.bottleType " +
```

Comment:
Niesom si isty ci tento riadok treba, skusal si tu metodu ci funguje tak ako predpokladas? Takisto neviem ci funguje to predavanie parametrov. Co som videl ja zatial tak sa pouzivalo ?1 ?2 a nie :meno, ale kludne to moze fungovat aj takto, skus si na to spravit nejaky test. Inak to vyzera super, dobra praca.